templates/ackee.ts

```diff
@@ -0,0 +1,86 @@
+import {
+  AppService,
+  createTemplate,
+  randomPassword,
+  MongoService,
+} from "~templates-utils";
+
+export default createTemplate({
+  name: "Ackee",
+  schema: {
+    type: "object",
+    required: ["projectName", "domain", "ackeeUsername", "ackeePassword", "appServiceName", "mongoServiceName"],
+    properties: {
+      projectName: {
+        type: "string",
+        title: "Project Name",
+      },
+      domain: {
+        type: "string",
+        title: "Domain",
+      },
+      ackeeUsername: {
+        type: "string",
+        title: "Ackee Username",
+      },
+      ackeePassword: {
+        type: "string",
+        title: "Ackee Password",
+      },
+      appServiceName: {
+        type: "string",
+        title: "App Service Name",
+        default: "ackee",
+      },
+      mongoServiceName: {
+        type: "string",
+        title: "MongoDB Service Name",
+        default: "mongodb",
+      },
+    },
+  } as const,
+  generate({
+    projectName,
+    domain,
+    ackeeUsername,
+    ackeePassword,
+    appServiceName,
+    mongoServiceName,
+  }) {
+    let mongoPassword = randomPassword();
+
+    const appService: AppService = {
+      projectName,
+      serviceName: appServiceName,
+      env: [
+        `WAIT_HOSTS=${projectName}_${mongoServiceName}:27017`,
+        `ACKEE_MONGODB=mongodb://mongo:${mongoPassword}@${projectName}_${mongoServiceName}:27017`,
+        `ACKEE_USERNAME=${ackeeUsername}`,
+        `ACKEE_PASSWORD=${ackeePassword}`,
+      ].join("\n"),
+      source: {
+        type: "image",
+        image: "electerious/ackee",
+      },
+      proxy: {
+        port: 3000,
+        secure: true,
+      },
+      domains: [{ name: domain }],
+    };
+
+    const mongoService: MongoService = {
+      projectName,
+      serviceName: mongoServiceName,
+      image: "mongo:4.4.13",
```

Comment:
Do you have a special reason for setting the version like this? I think it's better to keep `mongo:4`. It will still get the latest `4.*` but it's pinned to the major version.

Reply:
Yess, Just noticed it too, thanks!
Fixed in [709012d](https://github.com/easypanel-io/templates/pull/10/commits/709012df98cc342bc90d10516e43ff17eb887362)